scene/resources/material.cpp

```diff
@@ -80,13 +81,24 @@ void Material::_validate_property(PropertyInfo &property) const {
 	}
 }
 
+void Material::inspect_native_source() {
```

Comment:
This seems unnecessary. Shouldn't this be contained within the editor class?

This should be in a similar place as "Convert to Shader Material"

Reply:
there are no plugins of this type, this interface kind of sucks but its either this or creating a new class of editor plugins

Reply:
How much work would it be to create a new class of editor plugins? Because right now the code is not ideal and the UX is not ideal. Users will expect this feature to be alongside "Convert to ShaderMaterial" and not in the tools menu of the resource.